gui/workorder-recheck.lua

```diff
@@ -0,0 +1,22 @@
+-- Makes selected `Active` work order `Checking`.
```

Comment:
Maybe this?
```suggestion
-- 
```

(in the changelog, it would look like this:)
```
```suggestion
`workorder`: resets the selected work order to the ``Checking`` state
```

Reply:
(not a native speaker here, so yeah :) )